lib/XML/Sig.pm

```diff
@@ -1437,6 +1449,31 @@ sub _load_rsa_key {
     }
 }
 
+##
+## _load_hmac_key_info()
+##
+## Arguments:
+##    none
+##
+## Returns: nothing
+##
+## Populate:
+##   self->{KeyInfo}
+##
+sub _load_hmac_key_info {
+    my $self = shift;
+
+    if (! defined $self->{ key_name }) {
+        return;
+    }
+
+    $self->{KeyInfo} = <<"KEYINFO";
+<dsig:KeyInfo>
```

Comment:
Just look and feel: I find the indenting a bit odd here

Reply:
I was trying the put the indenting correct for the XML fragment.  It might be better if I do it differently